File: 80-remove-duplicates-from-sorted-array-ii/remove-duplicates-from-sorted-array-ii.c
int removeDuplicates(int* nums, int numsSize) 
{
    int F = 1;
    int R = 1;
    int counter = 1;
    while(F < numsSize)
    {
        if(nums[F] == nums[F - 1])
        {
            counter++;
        }
        else
        {
            counter = 1;
        }
        if(counter <= 2)
        {
            nums[R] = nums[F];
            R++;
        }
        F++;
    }    
    return R;
}

File: 80-remove-duplicates-from-sorted-array-ii/remove-duplicates-from-sorted-array-ii.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        Front = 1
        Rear = 1
        count = 1

        while Front < len(nums):
            if nums[Front] == nums[Front - 1]:
                count += 1
            else:
                count = 1

            if count <= 2:
                nums[Rear] = nums[Front]
                Rear +=1
            Front += 1
        return Rear
        